mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -403,6 +530,35 @@ def __call__(self, results):
                 target, target_weight = self._megvii_generate_target(
                     results['ann_info'], joints_3d, joints_3d_visible,
                     self.kernel)
+        elif self.encoding == 'UDP':
+            if self.target_type == 'CombinedTarget':
```

Comment:
`assert target_type in ['GaussianHeatMap', 'CombinedTarget']`

delete line 540 and 541